README.md

```diff
@@ -87,11 +87,11 @@ For this you can use `captureAndSave` method by passing directory location. By d
 
 ```dart
 final directory = (await getApplicationDocumentsDirectory ()).path; //from path_provide package
-String fileName = DateTime.now().microsecondsSinceEpoch;
+String fileName = DateTime.now().microsecondsSinceEpoch as String;
```

Comment:
Actually this won't work either, because you can't cast `int` to `String`. One solution would be this:
```suggestion
String fileName = '${DateTime.now().microsecondsSinceEpoch}';
```

Reply:
this seems better approach!